src/AppInstallerCommonCore/Public/winget/UserSettings.h

```diff
@@ -106,6 +157,8 @@ namespace AppInstaller::Settings
         }
 
         static std::filesystem::path SettingsFilePath();
+        static std::tuple<std::string, std::string, std::string> GetFeatureInfo(ExperimentalFeature feature);
+        static std::vector<std::tuple<std::string, std::string, std::string>> GetFeaturesInfo();
```

Comment:
>static std::vector<std::tuple<std::string, std::string, std::string>> GetFeaturesInfo(); [](start = 8, length = 88)

I don't really agree with tacking the features directly into the UserSettings.  It is definitely a layer on top of them, and so it should be a separate type.